node/src/index.ts

```diff
@@ -459,6 +459,73 @@ export class LocalTable<T = number[]> implements Table<T> {
   async delete (filter: string): Promise<void> {
     return tableDelete.call(this._tbl, filter).then((newTable: any) => { this._tbl = newTable })
   }
+
+  /**
+   * Clean up old versions of the table, freeing disk space.
+   *
+   * @param olderThan The minimum age in minutes of the versions to delete. If not
+   *                  provided, defaults to two weeks.
+   * @param deleteUnverified Because they may be part of an in-progress
+   *                  transaction, files newer than 7 days old are not deleted
+   *                  by default. If you are sure that there are no in-progress
+   *                  transactions, then you can set this to `true` to delete all
+   *                  files older than `older_than`.
+   * @returns
+   */
+  async cleanup_old_versions (olderThan?: number, deleteUnverified?: boolean): Promise<CleanupStats> {
+    return tableCleanupOldVersions.call(this._tbl, olderThan, deleteUnverified)
+  }
+
+  async compact_files (options?: CompactionOptions): Promise<CompactionMetrics> {
+    const optionsArg = (options != null) || {}
```

Comment:
```suggestion
    const optionsArg = options || {}
```
This will use `options` unless it is `null` or `undefined`.  The way it is currently written `optionsArg` will either be given the value `{}` or the value `(options != null)`, which is `true`.